examples/defaulter-gen/Makefile

```diff
@@ -0,0 +1,15 @@
+TOOL=defaulter-gen
+
+test:
+	@if ! git diff --quiet HEAD; then \
+	    echo "FAIL: git client is not clean"; \
+	    false; \
+	fi
+	@go build -o /tmp/$(TOOL)
+	/tmp/$(TOOL) --logtostderr --v=6 -i ./output_tests/wholepkg -O zz_generated
+	@if ! diff -q ./output_tests/wholepkg/zz_generated.expected ./output_tests/wholepkg/zz_generated.go; then \
```

Comment:
blech wrt having external deps.  OK.  Can you comment that?

The point of checking it in is so that a human can look at a series of trivial output tests, and comprehend the output. --verify-only doesn't give me that?